lib/wicked/IPv6.pm

```diff
@@ -0,0 +1,133 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Helper module for IPv6 tests
+# Maintainer: Jose Lausuch <jalausuch@suse.com>
+
+package wicked::IPv6;
+
+use Mojo::Base 'wickedbase';
+use testapi;
+
+sub evaluate_step {
+    my ($self, $cmd, $value, $expected) = @_;
+    my $output = script_output($cmd);
+    record_info('output', "cmd: $cmd \noutput:\n$output");
+    die("$value is not expected") if ($expected == 0 && $output =~ $value);
```

Comment:
just realized - in my version all checks will happen nevertheless something is failing and only at the end there is die . I think we need to keep this behavior

Reply:
I changed this behaviour now. Everything will be checked and `die` at the end if something fails.